scipp-imaging-2D.ipynb

```diff
@@ -0,0 +1,8350 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to start\n",
+    "\n",
+    "Before starting you must ensure that `scipp` and `mantid` are on your `PYTHONPATH`.\n",
+    "\n",
+    "For `scipp` and `mantid` follow instructions at: https://scipp.readthedocs.io/en/latest/getting-started/installation.html.\n",
+    "\n",
+    "Additionally, the other package dependencies are `fabio` and `astropy`: `conda install fabio astropy`\n",
+    "\n",
+    "This notebook follows the steps and uses the parameters from this reduction script https://git.esss.dk/testbeamline/gp2/blob/master/reduce.py but have them converted using equivalent `scipp` functionality.\n",
+    "\n",
+    "Before starting you need to configure:\n",
+    "\n",
+    "- The experiment directory in the `experiment_dir` variable - this "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "# Specify where the data can be found\n",
+    "experiment_dir = \"/home/dtasev/dev/scipp_imaging/RAL_Mantid_Sep2018/\"\n",
+    "\n",
+    "# let's get the process started:\n",
+    "tofs_path = os.path.join(experiment_dir, 'data_GP2', 'metadata', 'GP2_BCC_time_values.txt')\n",
+    "sample_path = os.path.join(experiment_dir, 'data_GP2', 'Timeslices WFM BBC Steel')\n",
+    "ob_path = os.path.join(experiment_dir, 'data_GP2', 'Timeslices WFM Open Beam')\n",
+    "\n",
+    "# defining grouping of 2D detector pixels\n",
+    "grouping_number = 3\n",
+    "nx_target = grouping_number\n",
+    "ny_target = grouping_number\n",
+    "\n",
+    "frame_parameters = [(15167,23563,64),\n",
+    "                    (24393,32758,64),\n",
+    "                    (33365,40708,64),\n",
+    "                    (41410,48019,64),\n",
+    "                    (49041,55311,64),\n",
+    "                    (56077,59872,64)]\n",
+    "\n",
+    "# Two options availabe for frame shift increments: 1) Robins Calibrated Values 2) Owens Calculated Values.\n",
+    "\n",
+    "frame_shift_increments = [-6630.0, -2420.0, -2253.0, -2095.0, -1946.0, -1810.0]\n",
+    "# frame_shift_increments = [-6630,-2423,-2252,-2058,-1949,-1576]\n",
+    "frame_shifts = [sum(frame_shift_increments[:i + 1]) for i in range(len(frame_shift_increments))]\n",
+    "\n",
+    "# rebin region reduced to cut off frames that contain no data\n",
+    "rebin_parameters = (8500, 43000, 64)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# created by Owen Arnold (optional e-mail) on 2018-09-19\n",
```

Comment:
This is not correct, but you might reference the original repository and then set yourself as the author of the present script.